src/core/datastore/cachestore.js

```diff
@@ -81,14 +76,14 @@ export class CacheStore extends NetworkStore {
    *                                                                            from the local cache.
    * @return  {Promise}                                                         Promise
    */
-  pendingSyncCount(query, options) {
+  pendingSyncCount(query) {
```

Comment:
You should remove the information about the `options` in the documentation for the function.

Reply:
If there's nothing more to change on this PR, can we do this in a later PR? I'd like to avoid rebasing 3 branches (and complicating the review of the other two open PRs) because of removing a comment. I'll do it in part 4 (the one I'm working on right now), how does that sound?

Reply:
That is fine with me.